tests/rptest/tests/tx_abort_index_test.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2022 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import re
+
+from rptest.clients.types import TopicSpec
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.cluster import cluster
+from rptest.services.redpanda import RESTART_LOG_ALLOW_LIST
+from rptest.services.redpanda_installer import RedpandaInstaller, wait_for_num_versions
+from rptest.services.redpanda import RedpandaService
+from ducktape.utils.util import wait_until
+from time import sleep
+
+from confluent_kafka import (Producer, KafkaException)
+from random import choice
+from string import ascii_uppercase
+
+
+def on_delivery(err, msg):
+    if err is not None:
+        raise KafkaException(err)
+
+
+TOPIC = "topic1"
+PAYLOAD_1KB = ''.join(choice(ascii_uppercase) for i in range(1024))
+
+
+class TxAbortSnapshotTest(RedpandaTest):
+    topics = [TopicSpec(name=TOPIC)]
```

Comment:
remove `TOPIC` and allow a random name to be generated

Reply:
why, I'm using the name and I need to know it

Reply:
because it generates a name for you automatically.

```
topics = [TopicSpec()]
```

Then in your test, the name is available to you by calling `self.topic`

Reply:
done